src/components/dashboard/main-content/course-enrollments/data/tests/selectors.test.js

```diff
@@ -0,0 +1,31 @@
+import {
+  filterCourseRuns,
+} from '../selectors';
+import { createRawCourseRun } from '../../tests/enrollment-testutils';
+
+describe('getCourseRunsByStatus', () => {
+  it('categorizes in_progress and upcoming courses correctly', () => {
```

Comment:
maybe go with 'test' instead of 'it' for cross-file consistency? not sure which one we standardized on

Reply:
Yeah, I just do `it` on autopilot, I think most of ours are `test`, so I'll double check and err on the side of consistency.

Reply:
Actually, I'm seeing more its than tests in the current code. I think since I moved the other tests into a describe block I'm gonna change them to it as well.

Reply:
yeah this is non blocking so whichever makes more sense yeah let's use that one.